tests/ctest/test_opmphm_predictor.c

```diff
@@ -0,0 +1,195 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see doc/LICENSE.md or https://www.libelektra.org)
+ */
+
+#include "../../src/libs/elektra/opmphmpredictor.c"
```

Comment:
Maybe its better to adopt the include path. `""` are also not really portable.

Reply:
How should I adopt the include path? Without `"` is is not working

Reply:
https://cmake.org/cmake/help/v3.0/command/include_directories.html

`include_directories ("${CMAKE_SOURCE_DIR}/src/src/libs/elektra")` in the CMakeLists.txt and `#include <opmphmpredictor.c>` should work.